src/components/ModalNewRequeriments/index.js

```diff
@@ -0,0 +1,243 @@
+import React, { useRef, useState } from "react";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import CloseIcon from "@material-ui/icons/Close";
+import AddIcon from "@material-ui/icons/Add";
+import CameraAltOutlinedIcon from "@material-ui/icons/CameraAltOutlined";
+import DoneOutlineOutlinedIcon from "@material-ui/icons/DoneOutlineOutlined";
+import { DropzoneDialog } from "material-ui-dropzone";
+import Modal from "./Modal";
```

Comment:
Aqui seria assim, não?
```suggestion
import Modal from "../Modal";
```